packages/plugins/actions/actions-core/src/actions/ScrollTo.test.js

```diff
@@ -84,7 +92,7 @@ test('ScrollTo with no params', async () => {
   });
   const button = context._internal.RootBlocks.map['block:root:button:0'];
   button.triggerEvent({ name: 'onClick' });
-  expect(mockWindowScrollTo.mock.calls).toEqual([[{}]]);
```

Comment:
This should not be removed, this means `mockWindowScrollTo ` was never called, where the test is testing that it should

Reply:
We should change to something like:
```js
  expect(() => button.triggerEvent({ name: 'onClick' })).toThrow(
    'Invalid ScrollTo, check action params. Received "undefined"'
  );
```
since action will now throw due to invalid params